spec/02-integration/01-cmd/02-start_stop_spec.lua

```diff
@@ -116,9 +153,13 @@ describe("kong start/stop", function()
     end)
     it("start inexistent prefix", function()
       local ok, _, stdout, stderr = exec "start --prefix foobar"
-      assert.False(ok)
-      assert.equal("", stdout)
-      assert.matches("foobar does not exist", stderr, nil, true)
+      assert.True(ok)
+      assert.not_equal("", stdout)
+      assert.equal("", stderr)
+      finally(function()
+        helpers.execute(KILL_ALL)
+        helpers.dir.rmtree("foobar")
+      end)
```

Comment:
Please be more careful, this test clearly belongs to a category of "errors" test, as stated by the scoping `describe`. If you change the behavior of the CLI (I won't even discuss the legitimacy of automatically creating a non-existing folder on behalf of the user, that is unexpected behavior to me), this is not testing the opposite. This test should be moved outside of the errors.